modules/core/portal-bootstrap/system.packages.extra.bnd

```diff
@@ -290,12 +290,12 @@ Provide-Capability:\
 	@lib/com.liferay.petra.mail.jar,\
 	@lib/com.liferay.petra.memory.jar,\
 	@lib/com.liferay.petra.nio.jar,\
-	@lib/com.liferay.petra.portlet.url.builder.jar,\
 	@lib/com.liferay.petra.process.jar,\
 	@lib/com.liferay.petra.reflect.jar,\
 	@lib/com.liferay.petra.sql.dsl.api.jar,\
 	@lib/com.liferay.petra.sql.dsl.spi.jar,\
 	@lib/com.liferay.petra.string.jar,\
 	@lib/com.liferay.petra.url.pattern.mapper.jar,\
-	@lib/com.liferay.petra.xml.jar
+	@lib/com.liferay.petra.xml.jar,\
```

Comment:
@julschong The commit message is not consistent with the change in this commit, please update.

Tina